### PR TITLE
Motion Amendments 2019 - Committe/Team Senior Member

### PR DESCRIPTION
**Pull request by ChrisWrightWCA**
Changes to the motions to recognize the new Committee/Team Senior Member role

### CODE REVIEW

File: documents/motions/02.2018.1 - Definitions.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Within WCA we use the following definitions for terminology.
     </tr>
     <tr>
       <td>Committee</td>
-      <td>A committee of the WCA that has been appointed by the WCA under the terms of the WCA Bylaws or any of the Motions. Leaders of committees are WCA Staff and rest of committees members are WCA External Staff.</td>
+      <td>A Committee of the WCA that has been appointed by the WCA under the terms of the WCA Bylaws or any of the Motions. Leaders and Senior Members of Committees are WCA Staff and rest of Committee members are WCA External Staff.</td>
```
Comment (jfly) [inline documents/motions/02.2018.1 - Definitions.md]: The definition of "Committee" uses the word "Committee"?

File: documents/motions/02.2018.1 - Definitions.md [hunk 2]
```diff
@@ -137,7 +142,7 @@ Within WCA we use the following definitions for terminology.
     </tr>
     <tr>
       <td>Team</td>
-      <td>A team of the WCA that has been appointed by the WCA under the terms of these Bylaws or any of the Motions. Leaders of teams are WCA Staff and rest of team members are WCA External Staff.</td>
+      <td>A team of the WCA that has been appointed by the WCA under the terms of these Bylaws or any of the Motions. Leaders and Senior Members of teams are WCA Staff and rest of team members are WCA External Staff.</td>
```
Comment (jfly) [inline documents/motions/02.2018.1 - Definitions.md]: nit: In the definition of "Committee" earlier, you capitalized the `c`, do  you want to capitalize the the two `t`s here?

Also seems weird to be using the word "team" to define the term "team" here.

File: documents/motions/10.2017.0 - Committees and Teams.md [hunk 3]
```diff
@@ -32,4 +32,9 @@ The WCA will have Committees and/or Teams that have the role of Advisory Committ
       3. The WCA Board of Directors shall have the right to withdraw the appointment of a Committee Leader, if the Committee Leader does not comply with the rights and duties of Committee Leader, after advisory consultation by the WCA Ethics Committee.
          1. For the withdrawal of the WCA Ethics Committee Leader, the WCA Board of Directors will ask for advisory consultation to the WCA Disciplinary Committee.
          2. If none of these consultations is possible due to the inexistence of these Committees or because these Committees are not functional, the WCA Board of Directors will proceed without consultation.
-5. All members of the Committee shall communicate with a carbon copy (CC) to all the other members of the Committee.
+5. Each WCA Committee and Team may have one or more Senior Member(s).
+   1. Senior Members are eligible for promotion after they have served at least 1 year in at least one of the Committees.
+      1. If a Member joins a new Team and lefts the old one, the time spent in the old Team shall count for the 1 year period.
```
Comment (jfly) [inline documents/motions/10.2017.0 - Committees and Teams.md]: lefts -> leaves

maybe: "count for" -> "count towards"

File: documents/motions/10.2017.0 - Committees and Teams.md [hunk 4]
```diff
@@ -32,4 +32,9 @@ The WCA will have Committees and/or Teams that have the role of Advisory Committ
...
+   2. Leaders shall appoint Members for promotion to become Senior Members who are doing and outstanding work for their Team and for the WCA.
```
Comment (jfly) [inline documents/motions/10.2017.0 - Committees and Teams.md]: This reads weird to me. It sounds like it's saying that Members who aren't doing outstanding work would be appointed to become Senior Members who are now doing outstanding work?

I would reorder this a bit. How about:

- Leaders shall (will? can? may?) appoint Members who are doing outstanding work to the role of Senior Member.

File: documents/motions/02.2018.1 - Definitions.md [hunk 5]
```diff
@@ -114,6 +114,10 @@ Within WCA we use the following definitions for terminology.
       <td>Senior Delegate</td>
       <td>A WCA Staff Member and Full Delegate who has been appointed by the WCA to manage a group of WCA Delegates.</td>
     </tr>
+    <tr>
+      <td>Senior Member</td>
+      <td>A member of one or more Committee(s) or Team(s) who has been a member in at least one of the Committee(s) Team(s) for a year and has been appointed by its Leader.</td>
```
Comment (AlbertoPdRF) [inline documents/motions/02.2018.1 - Definitions.md]:
```suggestion
      <td>A Committee or Team Member who has been on it for at least a year and has been appointed by its Leader.</td>
```

I think this looks clearer. Also, in my opinion, a Senior member should be a per-Committee/Team role, so I suggested accordingly.

File: documents/motions/02.2018.1 - Definitions.md [hunk 6]
```diff
@@ -128,6 +132,7 @@ Within WCA we use the following definitions for terminology.
           <li>Committee/Team Leader</li>
           <li>Senior Delegate</li>
           <li>Full Delegate</li>
+          <li>Senior Member of a Committee/Team</li>
```
Comment (AlbertoPdRF) [inline documents/motions/02.2018.1 - Definitions.md]: The order here seems weird to me.

File: documents/motions/02.2018.1 - Definitions.md (hunk 2 above)
Comment (AlbertoPdRF) [inline documents/motions/02.2018.1 - Definitions.md]:
```suggestion
      <td>A Team of the WCA that has been appointed by the WCA under the terms of these Bylaws or any of the Motions. Leaders and Senior Members of Teams are WCA Staff and rest of Team Members are WCA External Staff.</td>
```

File: documents/motions/10.2017.0 - Committees and Teams.md [hunk 7]
```diff
@@ -32,4 +32,9 @@ The WCA will have Committees and/or Teams that have the role of Advisory Committ
       3. The WCA Board of Directors shall have the right to withdraw the appointment of a Committee Leader, if the Committee Leader does not comply with the rights and duties of Committee Leader, after advisory consultation by the WCA Ethics Committee.
          1. For the withdrawal of the WCA Ethics Committee Leader, the WCA Board of Directors will ask for advisory consultation to the WCA Disciplinary Committee.
          2. If none of these consultations is possible due to the inexistence of these Committees or because these Committees are not functional, the WCA Board of Directors will proceed without consultation.
-5. All members of the Committee shall communicate with a carbon copy (CC) to all the other members of the Committee.
+5. Each WCA Committee and Team may have one or more Senior Member(s).
```
Comment (AlbertoPdRF) [inline documents/motions/10.2017.0 - Committees and Teams.md]: I think this whole section should go. Instead, we should add a line under 4.2.2. similar to it. E.g. "Promoting outstanding Committee Members to Committee Senior Members."

Comment (AlbertoPdRF) [reply documents/motions/10.2017.0 - Committees and Teams.md]: Add "promoting and demoting" to 4.2.2.

File: documents/motions/10.2017.0 - Committees and Teams.md [hunk 8]
```diff
@@ -32,4 +32,9 @@ The WCA will have Committees and/or Teams that have the role of Advisory Committ
...
+   1. Senior Members are eligible for promotion after they have served at least 1 year in at least one of the Committees.
```
Comment (AlbertoPdRF) [inline documents/motions/10.2017.0 - Committees and Teams.md]:
```suggestion
   1. Senior Members are eligible for promotion after they have served at least 1 year, cumulatively, as a Staff and/or External Staff Member.
```

**Approved by the Board**

File: documents/motions/10.2017.0 - Committees and Teams.md (hunk 3 above)
Comment (AlbertoPdRF) [reply documents/motions/10.2017.0 - Committees and Teams.md]: Remove this point.

**Approved by the Board**

File: documents/motions/10.2017.0 - Committees and Teams.md (hunk 4 above)
Comment (AlbertoPdRF) [inline documents/motions/10.2017.0 - Committees and Teams.md]:
```suggestion
   2. A Committee Leader may give a member the status of Senior Member if the Leader judges that the member has shown their fitness for the position.
```

**Approved by the Board**

File: documents/motions/10.2017.0 - Committees and Teams.md [hunk 9]
```diff
@@ -32,4 +32,9 @@ The WCA will have Committees and/or Teams that have the role of Advisory Committ
...
+      1. If a Member joins a new Team and lefts the old one, the time spent in the old Team shall count for the 1 year period.
+   2. Leaders shall appoint Members for promotion to become Senior Members who are doing and outstanding work for their Team and for the WCA.
+   3. Senior Members are automatically Staff Members, therefore they are eligible for Board Elections and are Voting Members as well.
```
Comment (AlbertoPdRF) [inline documents/motions/10.2017.0 - Committees and Teams.md]: Remove this point.

**Approved by the Board**

**Comment by AlbertoPdRF** [comment]
This one already includes the changes approved by the Board and all the commits have been squashed into one. Someone @thewca/board please review it!

File: documents/motions/02.2018.1 - Definitions.md (hunk 1 above)
Comment (cubewhiz) [reply documents/motions/02.2018.1 - Definitions.md]: I think this is okay. It's not so much the definition of committee that is important, as much as to make it clear that when we use the word committee, this is what we are referring to.

**Comment by AlbertoPdRF** [comment]
I've rebased this PR after having merged #32 and I'm merging it now!

**Pull request merged**